client/packages/system/src/Stock/Components/Ledger/useLedgerColumns.ts

```diff
@@ -0,0 +1,108 @@
+import {
+  ColumnFormat,
+  SortBy,
+  useColumns,
+  useFormatDateTime,
+  useTranslation,
+} from '@openmsupply-client/common';
+import { LedgerRowFragment } from '../../api';
+
+type MovementType =
+  | 'INBOUND_SHIPMENT'
+  | 'OUTBOUND_SHIPMENT'
+  | 'INBOUND_RETURN'
+  | 'OUTBOUND_RETURN'
+  | 'PRESCRIPTION'
+  | 'INVENTORY_ADDITION'
+  | 'INVENTORY_REDUCTION';
+
+export enum ColumnKey {
+  'DateTime' = 'datetime',
+  'Time' = 'time',
+  'Name' = 'name',
+  'Quantity' = 'quantity',
+  'Type' = 'type',
+  'Reason' = 'reason',
+}
```

Comment:
Since I've used these in `useStockLedger` for mapping column keys to GraphQL sort keys, I thought I'd make it an enum to ensure legal values are enforced.